src/main/frontend/format/block.cljs

```diff
@@ -365,6 +385,7 @@
                      :end-pos (or first-block-start-pos
                                   (utf8/length encoded-content))}
               :body (take-while (fn [block] (not (heading-block? block))) blocks)
+              :ref-pages (extract-filetags content)
```

Comment:
Don't think it's the 'right' place for this. In particular, don't like that it's parsing properties all over again, even though they are already parsed in `:page/properties`, but I couldn't quite figure out how to access them from here. Most likely adding filetags should happen somewhere else?